Cubical/Categories/Monoidal/Closed.agda

```diff
@@ -0,0 +1,60 @@
+-- Closed Monoidal Categories
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Monoidal.Closed where
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Constructions.BinProduct
+open import Cubical.Categories.Functor.Base
+open import Cubical.Categories.Functor.BinProduct
+open import Cubical.Categories.Morphism
+open import Cubical.Categories.NaturalTransformation.Base
+open import Cubical.Foundations.Prelude
+open import Cubical.Categories.Monoidal.Base
+open import Cubical.Categories.Monoidal.Symmetric
+open import Cubical.Categories.Adjoint
+
+open UnitCounit
+
+module _ {ℓ ℓ' : Level} (C : Category ℓ ℓ') where
+  open Category C
+  open Functor
+
+  -- the closed structure
+  record ClosedStr : Type (ℓ-max ℓ ℓ') where
```

Comment:
Perhaps "closed structure" / `ClosedStr` is not the optimal name for just a bifunctor `[─,─]`, since there is also this notion:
https://ncatlab.org/nlab/show/closed+category

What about `InternalHomStr` or `BifunctorStr`?